src/extensions/renderer/canvas/index.js

```diff
@@ -63,13 +64,28 @@ function CanvasRenderer( options ){
   container.appendChild( r.data.canvasContainer );
 
   if( (container.getAttribute('style') || '').indexOf(tapHlOff) < 0 ){
-    container.setAttribute( 'style', ( container.getAttribute( 'style' ) || '' ) + tapHlOff );
+    container.style[tapHlOffAttr] = tapHlOffStyle;
```

Comment:
The if check isn't necessary anymore if we're setting the properties in a keyed way

Reply:
check removed in b949970787f1b1ca522d23ff40f7093c1d608004